src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  margin: 8px;
+}
+
+.search-form {
+  margin: 20px auto 0;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 70px;
+}
+
+.search-input {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  background-image: url(./images/icon-search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  background-size: 19px 19px;
+  padding-left: 62px;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.form-small {
+  width: 100%;
+  height: 42px;
+}
+
+.input-small {
+  background-position: 13px center;
+  background-size: 11px 11px;
+  line-height: 20px;
+  padding-left: 33px;
+}
+
+.search-input::placeholder {
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.input-small::placeholder {
+  font-size: 14px;
+  padding-bottom: 11px;
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.input-small .search-input:focus {
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+}
+
+.search-input:focus::placeholder {
```

Comment:
you don't need to set a placeholder for `focus`. just set styles for `.search-input::placeholder`